docs: initial accessibility section in each component page (#175)
Refinitiv/refinitiv-ui: documents/src/pages/elements/combo-box.md

| ----------------- | ------------------------ |
| --list-max-width  | Max width of popup list  |
| --list-max-height | Max height of popup list |

## Accessibility
::a11y-intro::

> `ef-combo-box` is assigned `role="combobox"` and can include properties such as `aria-autocomplete`, `aria-controls`, `aria-expanded` and `aria-activedescendant`. The expanded List Box is assigned `role="listbox"` while the List Box Item is assigned `role="option"` and can include properties such as `aria-label` and `aria-selected`. The element’s state programmatically updates to match its visual state.  

Reviewer: ```suggestion
`ef-combo-box` is assigned `role="combobox"` and can include properties such as `aria-autocomplete`, `aria-controls`, `aria-expanded` and `aria-activedescendant`. The expanded list box is assigned `role="listbox"` while its items are assigned `role="option"` and `aria-selected` which depends on item's selection state.
```
